### PR TITLE
Replace yarn with pnpm

### PR DESCRIPTION
**Pull request by sz-piotr**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,18 +16,18 @@
...
     "watch:dependencies": "turbo watch build --filter=@l2beat/frontend^... --filter=@l2beat/backend^...",
     "clean": "turbo run clean",
-    "fix": "yarn lint:fix && yarn format:fix",
-    "format": "wsrun -ecam format",
-    "format:fix": "wsrun -ecam format:fix",
-    "lint": "wsrun -ecam lint",
-    "lint:fix": "wsrun -ecam lint:fix",
-    "start": "cd packages/backend && yarn start",
+    "fix": "pnpm lint:fix && pnpm format:fix",
+    "format": "pnpm -r format",
```
Comment (torztomasz) [inline package.json:20]: let's remove wsrun from deps then

**Review by torztomasz — CHANGES_REQUESTED** [review]

File: packages/protocolbeat/src/HomePage.tsx [hunk 2]
```diff
@@ -23,7 +23,7 @@ export function HomePage() {
         {result.data.map((entry, i) => (
           <li key={i}>
             <Link to={`/ui/p/${entry.name}`}>
-              {entry.name} ({entry.chains.length} chains)
+              {entry.name} ({entry.chains.length} chains2)
```
Comment (torztomasz) [inline packages/protocolbeat/src/HomePage.tsx:26]: intended?

Comment (imxeno) [reply packages/protocolbeat/src/HomePage.tsx:26]: not intended

File: .github/workflows/chromatic.yml [hunk 3]
```diff
@@ -13,14 +13,24 @@ jobs:
...
+
       - name: Install dependencies
-        run: yarn
+        run: pnpm install
       - name: Build dependencies
-        run: yarn build:dependencies
+        run: pnpm build:dependencies
       - name: Publish to Chromatic
         run: >
           npx chromatic
```
Comment (torztomasz) [inline .github/workflows/chromatic.yml:36]:
```suggestion
          pnpm exec chromatic
```

**Review by torztomasz — APPROVED** [review]

**Pull request merged**